features/breol_news/breol_news.strongarm.inc

```diff
@@ -59,6 +59,27 @@ function breol_news_strongarm() {
   $strongarm->value = 1;
   $export['comment_subject_field_breol_news'] = $strongarm;
 
+  $strongarm = new stdClass();
```

Comment:
Is these 3 not from the `diff` module and not `scheduler` related?

Reply:
Removed.